test/cases/libvirt.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -euo pipefail
 
+#
+# This script executes tools/libvirt_test.sh with the appropiate image type and boot type
```

Comment:
Well this is more like a helper script that runs `libvirt` test which is just creating a qcow2 image (openstack is basically the same, AFAIK there is no real difference), it then modifies them with cloud-init and verify that they boot.. Also it's testing UEFI where supported.

Reply:
I'm trying to create this comments a bit "future proof" so they don't get outdated fast. Do you think the purpose is not clear or do you want to add more details?